config/jobs/kubernetes-sigs/secrets-store-sync-controller/secrets-store-sync-controller-config.yaml

```diff
@@ -0,0 +1,68 @@
+presubmits:
+  kubernetes-sigs/secrets-store-sync-controller:
+  - name: pull-secrets-store-sync-controller-lint
+    cluster: eks-prow-build-cluster
+    decorate: true
+    decoration_config:
+      timeout: 10m
+    always_run: true
+    path_alias: sigs.k8s.io/secrets-store-sync-controller
+    branches:
+    - ^main$
+    - ^release-*
+    labels:
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20231113-7213ea5323-1.25
```

Comment:
hi! The new image has been released, maybe we should use the latest image
`gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240515-17c6d50e24-master`